src/digitalIdentity-frontend/src/app/pages/login-page/login-page.component.spec.ts

```diff
@@ -57,4 +72,42 @@ describe('LoginPageComponent', () => {
     let test_div = de.query(By.css('.test-card'));
     expect(test_div).toBeNull();
   });
+
+  it('should have an initial state like', () => {
+    expect(de.query(By.css('input'))).toBeDefined();
+    expect(component.formGroup.invalid).toBeTruthy();
+  });
+
+  it('empty input form is invalid', (() => {
+    let form = component.formGroup;
+    expect(fixture.debugElement.nativeElement.querySelector('input').innerHTML).toBe("");
```

Comment:
there are multiple input fields within the HTML. This does not test the intended behavior.